src/lib/server/blog/posts.ts

```diff
@@ -77,6 +77,7 @@ function cacheBlogPosts(output: any): BlogPost[] {
       lastEditedAt: lastEdited,
       number: id,
       bodyHTML: html,
+      bodyText: bodyText,
```

Comment:
Nit: Redundant destructor. 
```suggestion
      bodyText,
```

Reply:
done